creator-node/src/routes/tracks.js

```diff
@@ -959,6 +959,69 @@ router.get(
  * Gets a streamable mp3 link for a track by encodedId. Supports range request headers.
  * @dev - Wrapper around getCID, which retrieves track given its CID.
  **/
-router.get('/tracks/cidstream/:CID', contentAccessMiddleware, getCID)
+router.get(
+  '/tracks/cidstream/:CID',
+  contentAccessMiddleware,
+  async (req, res, next) => {
+    const libs = req.app.get('audiusLibs')
+    const redisClient = req.app.get('redisClient')
+    const delegateOwnerWallet = config.get('delegateOwnerWallet')
+
+    const trackId = req.query.trackId
```

Comment:
`trackId` is part of the signature sent from DN, not a query param. a version of this can still work if we set trackId on the request object in the middleware and then access it here.

Reply:
added a commit to handle